Assets/GothicVR/Scripts/Creator/Meshes/AbstractMeshCreator.cs

```diff
@@ -487,5 +499,29 @@ protected Material GetEmptyMaterial()
 
             return material;
         }
+
+        private Material GetWaterMaterial(PxMaterialData materialData)
+        {
+            var shader = Shader.Find(waterShader);
+            Material material = new Material(shader);
+
+            // FIXME - Here we need to see if we can change materialData.waveSpeed
+
+            switch (materialData.name)
```

Comment:
Please don't! This is hard coded and will prevent mods etc. let's properly check how to do.
1. Check on OpenGothic and ask Try/Luis
2. Let's define a default speed for now. Maybe just something slower.